### PR TITLE
add DELETE in error message

### PR DESCRIPTION
**Pull request by CoryWritesCode**
My native prettier config updated a few lines in that file. Let me know if you want them back the other way.

This PR should address #8

### CODE REVIEW

File: src/firefunctions/index.js [hunk 1]
```diff
@@ -29,7 +29,9 @@ export class FireFunctions {
       case TYPES.GET:
         return functions.https.onRequest((request, response) => {
           callbackWithCorsWrapped(() => {
-            const subref = isUndefined(request.query[idKey]) ? '' : `/${request.query[idKey]}`;
+            const subref = isUndefined(request.query[idKey])
```
Comment (tstreamDOTh) [inline src/firefunctions/index.js]: Please revert the changes due to formatter, rest looks good

**Comment by CoryWritesCode** [comment]
@tstreamDOTh should be good to go now!

**Comment by tstreamDOTh** [comment]
Awesome

**Comment by tstreamDOTh** [comment]
@CoryWritesCode  Thanks for making the update, please star the repo if you liked it ⭐️
